codalab/apps/web/static/js/worksheet/worksheet_interface.js

```diff
@@ -311,6 +311,31 @@ var Worksheet = React.createClass({
 
         var worksheet_display = this.state.editMode ? raw_display : items_display;
 
+        $.fn.editable.defaults.mode = 'inline';
+        $(document).ready(function() {
+          $('.editable-field').editable({
+            send: 'always',
+            type: 'text',
+            params: function(params) {
+              var data = {};
+              data['worksheet_uuid'] = ws_obj.state.uuid;
+              data['name'] = params['name'];
+              data['value'] = params['value'];
+              data['command'] = 'wedit -' + params['name'][0] + " '" + params['value'] + "'";
```

Comment:
This looks pretty dangerous to me.  Can we put this in a method in the worksheet utils class that handles making these commands?